Fix weird position of 'X' close button

guywarburg opened the pull request:
Associated Issue: #4688

### Summary of Changes

* remove `margin-top` from the container in order to fix vertical positioning. Fixed only positioning for the 'Watch Expression' panel.

Example test plan:

- [x] Verify that it is a unique class.
- [x] test the veiw and validate that the button is aligned.

#goodnessSquad

jasonLaster commented:
Nice!

jasonLaster commented:
can you attach some screenshots?

guywarburg commented:
Sure,
before:
![screen shot 2017-11-21 at 12 24 56 am](https://user-images.githubusercontent.com/24923164/33044666-77b7d26c-ce52-11e7-843c-2ab07b190511.png)

after:
![screen shot 2017-11-21 at 12 25 08 am](https://user-images.githubusercontent.com/24923164/33044675-7be0fbac-ce52-11e7-947e-4a12de53d0ea.png)

jasonLaster commented:
haha - hard to see the change...

jselmani commented:
It appears the blue box has been made slightly smaller and moved up a tad.  I must have missed this @jasonLaster

guywarburg commented:
Yes, it's a minor change - I removed 2px that were added as a margin-top and change the alignment. I added screenshots with the grid view, hope they are more helpful.

before:
![screen shot 2017-11-21 at 12 24 56 am](https://user-images.githubusercontent.com/24923164/33091205-101c1ea8-ceff-11e7-8bf9-1dfec0fafcf1.png)

after:
![screen shot 2017-11-21 at 12 25 08 am](https://user-images.githubusercontent.com/24923164/33091229-28a60f9c-ceff-11e7-889b-44bca6b7061d.png)

codehag commented:
Looks good, thanks!
